packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx

```diff
@@ -25,6 +25,6 @@ export const NavGroup: React.FunctionComponent<NavGroupProps> = ({
     <h2 className={css(styles.navSectionTitle)} id={id}>
       {title}
     </h2>
-    <ul className={css(styles.navSimpleList)}>{children}</ul>
+    <ul className={css(styles.navList)}>{children}</ul>
```

Comment:
This matches what's in Core on https://pf4.patternfly.org/documentation/core/components/nav .